Kickstarter-iOS/Views/Controllers/RootCommentsViewModel.swift

```diff
@@ -0,0 +1,37 @@
+import KsApi
+import Prelude
+import ReactiveExtensions
+import ReactiveSwift
+
+public protocol RootCommentsViewModelInputs {
+  /// Call when the view loads.
+  func viewDidLoad()
+}
+
+public protocol RootCommentsViewModelOutputs {
+  /// Delete this when beginning implementation
+  var viewDidLoadTestOutput: Signal<Bool, Never> { get }
```

Comment:
I think you've added this just to templatize the tests a bit but it's probably not needed, just an empty tests file would also do.

Reply:
Updated!